pennylane/ops/channel.py

```diff
@@ -211,7 +211,8 @@ class DepolarizingChannel(Channel):
     num_params = 1
     num_wires = 1
     par_domain = "R"
-    grad_method = "F"
+    grad_method = "A"
+    grad_recipe = ([[1, 0, 1], [-1, 0, 0]],)
```

Comment:
Oh! Did our change to the grad recipe format allow this?? That's awesome.

Reply:
So basically, the grad recipe is `f(1) - f(0)`?

Reply:
Yes! The general recipe is 
```
d E(p)[rho]/dp = 1/(p1-p2)[E(p1)[rho]-E(p2)[rho]]
```
where `p1` and `p2` are two parameters that do not depend on `p`. If we choose them to be `p1=1` and `p2=0` we get
`E(1)[rho]-E(0)[rho]`, which I think is the cleanest form of the expression

Reply:
It's funny, because my suggestion to include an additional coefficient for the parameter was on a complete whim 😆 I think it happened during a github comment

Reply:
@antalszava did check with me that this was general enough for the channel formulas!

Reply:
@trbromley helped out here to get the direction. :slightly_smiling_face: :wink: